package.json

```diff
@@ -88,5 +89,6 @@
6378 -73.86529,-67.14061 -169.12974,-72.14537 -8.93104,-0.46919 -17.71311,-0.71608 -26.30336,-0.6871 -83.03906,0.28014 -147.857502,23.11494 -149.509319,54.71024 -0.786577,15.04533 12.887399,29.634243 36.327769,41.741243' style='fill:none;stroke:%23b3b3b3;stroke-width:15;stroke-linecap:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1' id='path5961' /%3E%3Cpath inkscape:connector-curvature='0' d='m 246.94025,-249.74892 c -16.87219,-14.033 -32.69622,-21.56881 -45.40971,-20.61296 -3.38495,0.25449 -6.5155,1.09073 -9.42465,2.57662 -31.03092,15.84954 -21.11773,97.82121 22.19078,183.025916 43.3085,85.20471863 103.57031,141.41684 134.60123,125.567287 29.84195,-15.242267 21.7875,-91.625309 -17.39278,-173.234763' style='fill:none;stroke:%23b3b3b3;stroke-width:15;stroke-linecap:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1' id='path5963' /%3E%3C/g%3E%3Cg style='stroke:%23ffffff' id='g5898' transform='translate(-1.271761,1.2717614)'%3E%3Cpath id='path4' style='fill:none;stroke:%23ffffff;stroke-width:15;stroke-linecap:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1' d='m 375.71533,-197.78708 c 5.79237,-29.55271 2.55311,-51.9812 -11.13823,-60.89412 -4.56379,-2.97098 -10.03532,-4.27106 -16.19328,-4.03671 -33.25294,1.2655 -86.90786,47.16266 -130.74568,114.83137 -51.95594,80.199958 -70.41967,160.576206 -41.21145,179.590426 13.86472,9.025783 35.95803,2.627575 60.91755,-15.202059' inkscape:connector-curvature='0' /%3E%3Cpath id='path6' style='fill:none;stroke:%23ffffff;stroke-width:15;stroke-linecap:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1' d='m 402.3614,-66.637116 c 24.66003,-9.589669 39.82357,-22.715516 40.6117,-37.790424 1.8227,-34.86378 -73.86529,-67.14061 -169.12974,-72.14537 -8.93104,-0.46919 -17.71311,-0.71608 -26.30336,-0.6871 -83.03906,0.28014 -147.857502,23.11494 -149.509319,54.71024 -0.786577,15.04533 12.887399,29.634243 36.327769,41.741243' inkscape:connector-curvature='0' /%3E%3Cpath id='path8' style='fill:none;stroke:%23ffffff;stroke-width:15;stroke-linecap:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1' d='m 246.94025,-249.74892 c -16.87219,-14.033 -32.69622,-21.56881 -45.40971,-20.61296 -3.38495,0.25449 -6.5155,1.09073 -9.42465,2.57662 -31.03092,15.84954 -21.11773,97.82121 22.19078,183.025916 43.3085,85.20471863 103.57031,141.41684 134.60123,125.567287 29.84195,-15.242267 21.7875,-91.625309 -17.39278,-173.234763' inkscape:connector-curvature='0' /%3E%3C/g%3E%3C/g%3E%3Cg id='g6023'%3E%3Cpath id='path5965' style='fill:%23b3b3b3;fill-opacity:1;stroke:none;filter:url(%23filter6003)' d='m 292.90684,-107.76848 a 20.562887,20.612961 0 0 1 -41.12577,0 20.562887,20.612961 0 1 1 41.12577,0 z' inkscape:connector-curvature='0' /%3E%3Cpath inkscape:connector-curvature='0' d='m 290.90684,-109.76848 a 20.562887,20.612961 0 0 1 -41.12577,0 20.562887,20.612961 0 1 1 41.12577,0 z' style='fill:%23ffffff;fill-opacity:1;stroke:none' id='path10' /%3E%3C/g%3E%3C/g%3E%3C/svg%3E%0A"
```

Comment:
I used https://jakearchibald.github.io/svgomg/ to minify it before encoding (to 21.83% the size) 
```suggestion
    "icon": "data:image/svg+xml,%3Csvg viewBox='0 0 256 256' xmlns='http://www.w3.org/2000/svg' preserveAspectRatio='xMidYMid'%3E%3Cdefs/%3E%3Cdefs%3E%3ClinearGradient x1='50%25' y1='4%25' x2='50%25' y2='100%25' id='a'%3E%3Cstop stop-color='%23FDFDFD' offset='0%25'/%3E%3Cstop stop-color='%23CBCBCB' offset='100%25'/%3E%3C/linearGradient%3E%3ClinearGradient x1='50%25' y1='4%25' x2='50%25' y2='100%25' id='b'%3E%3Cstop stop-color='%23B2E198' offset='0%25'/%3E%3Cstop stop-color='%2304A171' offset='100%25'/%3E%3C/linearGradient%3E%3C/defs%3E%3Ccircle fill='url(%23a)' cx='128' cy='128' r='127.9'/%3E%3Ccircle fill='url(%23b)' cx='127.9' cy='128.1' r='116.9'/%3E%3Cpath d='M100 110l-4 6-3-11 12-1-5 6zm104 8c-5-4-10-8-18-11a172 172 0 00-64-12c17-20 32-30 41-30l5 1c5 3 6 11 4 23a4 4 0 008 2c4-20-2-29-7-32-3-2-6-3-10-3-14 1-34 16-52 39l-20 2-2-10c-2-13 0-23 5-25l3-1c4 0 11 3 18 9a4 4 0 005-7c-9-7-17-11-24-10l-5 1c-5 3-8 7-10 14v20l1 10-14 4c-18 7-22 16-22 22-1 6 2 14 18 23a4 4 0 004-8c-9-5-14-10-14-15 0-6 11-13 30-17l7 19c-7 13-12 25-14 36-2 8-3 15-2 21 1 7 4 11 8 14 2 2 5 3 9 3 6 0 13-3 23-10a4 4 0 10-5-7c-9 7-18 10-22 7-5-3-6-13-3-26l10-28 4 7c9 19 21 36 33 47 6 5 12 9 17 11 3 2 6 2 9 2l7-1c5-3 8-7 10-13l1-20c-2-15-8-33-16-51a4 4 0 00-8 3c8 18 14 35 15 49 1 13-1 22-5 24-5 3-14-1-24-11a163 163 0 01-38-56 211 211 0 0115-24 199 199 0 0168 12c12 5 19 11 19 17 0 5-6 9-16 13a4 4 0 003 8c17-6 21-15 21-21 1-5-2-10-6-14z' fill='%23F2F8F6'/%3E%3Cpath d='M119 129a9 9 0 1019 0 9 9 0 00-19 0' fill='%23F1F8F3'/%3E%3C/svg%3E"
```